rotkehlchen/tests/unit/decoders/test_thegraph.py

```diff
@@ -163,14 +170,185 @@ def test_thegraph_delegated_withdrawn(database, ethereum_inquirer):
             event_type=HistoryEventType.STAKING,
             event_subtype=HistoryEventSubType.REMOVE_ASSET,
             asset=A_GRT,
-            balance=Balance(amount=FVal('1003.70342593701668535')),
+            balance=Balance(amount=FVal(withdrawn_amount)),
             location_label=ADDY_USER,
-            notes=(
-                'Withdraw 1003.70342593701668535 GRT'
-                ' from indexer 0x6125eA331851367716beE301ECDe7F38A7E429e7'
-            ),
+            notes=f'Withdraw {withdrawn_amount} GRT from indexer {indexer_address}',
             counterparty=CPT_THEGRAPH,
-            address=ADDY_THEGRAPH,
+            address=CONTRACT_STAKING,
+        ),
+    ]
+    assert expected_events == events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('arbitrum_one_accounts', [[ADDY_USER_1_ARB]])
+def test_thegraph_delegate_arbitrum_one(database, arbitrum_one_inquirer):
+    tx_hash = deserialize_evm_tx_hash('0x3c846f305330969fb0ddb87c5ae411b4e9692f451a7ff3237b6f71020030c7d1')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=arbitrum_one_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp = TimestampMS(1713366299000)
+    gas_fees, approve_amount, delegate_amount, delegate_tax = '0.000003483187128', '1.009574848602990389', '24.9745', '0.1255'  # noqa: E501
+    indexer_address = '0xE13840A2E92e0Cb17A246609b432D0fA2e418774'
```

Comment:
Here and in other places in these tests.

```suggestion
    indexer_address = string_to_evm_address('0xE13840A2E92e0Cb17A246609b432D0fA2e418774')
```